docs/finder-content-item.md

```diff
@@ -56,6 +56,80 @@ A boolean. Required.
 
 Used to decide if the summaries for Documents should be displayed in the results list. It will truncate the summary at the end of the first sentence.
 
-## facets
+## `facets`
 
-__TO DO__
+An array of hashes. Optional.
+
+Facets describe the metadata that the Finder deals with. They can contain several keys:
+
+### `key`
+
+A string. Required.
+
+`snake_case` string which matches to the field being searched in Rummager.
+
+### `filterable`
+
+A boolean. Required.
+
+Specifies if the facet should have a matching Filter for the results.
+
+### `display_as_result_metadata`
+
+A boolean. Required.
+
+Specifies if the facet should be returned as metadata underneath each result.
+
+### `name`
+
+A string. Required if `filterable` is set to `true`.
+
+A label for the Filter on the left hand side.
```

Comment:
Perhaps something like:

Used to label the filter panel for the facet.

For date facets, it may be used to label the metadata shown for each result. (See also `short_name`)